FWCore/Framework/doc/EventSelector.cc-with-cerr-traces

```diff
@@ -13,7 +13,7 @@
 #include "FWCore/Utilities/interface/Algorithms.h"
 
 #include "boost/algorithm/string.hpp"
-#include "boost/regex.hpp"
+#include <regex>
```

Comment:
This file is obsolete. I suggest we let this change go in and then I remove the file (among others). Unless this PR needs to be modified for other reasons, in which case I'd ask to leave this file untouched (but only in that case).